.gitignore

```diff
@@ -10,11 +10,10 @@
 .build
 .omake*
 release
-bin
-version.ml
 *.native
 _build
 *.byte
 *.d.native
 *.p.byte
 *.sexp
+.merlin
```

Comment:
You add `.merlin` here but actually `.merlin` file is on repo already.

Reply:
Good point. I should delete the `.merlin` file in the repo. Now that we're using dune, it automatically generates the `.merlin` files for us, and so we shouldn't be committing them.